modules/controller/world_model_tracking.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+
+from .. import object_in_world
+
+
+class WorldModelTracking:
+    """
+    keeps track of the real world location of landing pads that
+    are to be visited or have already been vsisited
+    """
+
+    def __init__(self):
+        self.__unconfirmed_positives = []
+        self.__false_positives = []
+        self.confirmed_positive = None
+
+    def __similar(self,
```

Comment:
Make this static method.

Reply:
if I make this a static method and make the constant DISTANCE_THRESHOLD a variable, how do I access DISTANCE_THRESHOLD in the __similar method?

Reply:
Add another parameter for distance threshold, then pass it in as an argument when calling this method.